test/test_sparse.py

```diff
@@ -591,6 +591,7 @@ def test_sparse_bool(self, device, dtype):
         b = a.to_sparse().to_dense()
         self.assertEqual(a, b)
 
+    @unittest.expectedFailure         # cf https://github.com/pytorch/pytorch/issues/108667
```

Comment:
Nope. We have a decorator that xfails if run with dynamo. That's what you want.

Reply:
FWIW I did try this out locally and if that error were to change from `NotImplementedError` to something else, this all works fine.

Reply:
https://github.com/pytorch/pytorch/pull/107863/commits/8eabd14675241dd0a27b777ae8dd270b424c13ba `skipIfTorchDynamo`  as used elsewhere in this test suite

Reply:
@amjames yes, this is a small bug in TorchDynamo, that it should create a break properly and fallback to eager but it doesn't. If anyone from sparse wants to have a knock at this one it should be a good introductory PR.